README.md

```diff
@@ -8,13 +8,13 @@ Serde Rust Serialization Framework
 Serde is a powerful framework that enables serialization libraries to
 generically serialize Rust data structures without the overhead of runtime type
 information. In many situations, the handshake protocol between serializers and
-serializees can be completely optimized away, leaving Serde to perform roughly
+serializes can be completely optimized away, leaving Serde to perform roughly
```

Comment:
This was correct before -- serializee means "thing being serialized." See https://en.wiktionary.org/wiki/-ee#Suffix